src/components/map/MapComponent.vue

```diff
@@ -161,8 +186,8 @@ function centerOnPoint(coordinates: [number, number]): void {
   });
 }
 
-function manuSelect(e: AvailableLocation): void {
-  if (!e && selectedPoint.name !== '') {
+function manuSelect(e: AvailableLocation | null): void {
+  if (!e && selectedPoint.name !== '' || e == null) {

```

Comment:
It kinda looks like this is the same thing as
```suggestion
  if (!e) {
```
Because the only falsy value `e` can have is `null`, at least according to the type, and this evaluates to true only when  `e` is `null`. I didn't test it though.